components/wmtk_components/isotropic_remeshing/internal/IsotropicRemeshing.cpp

```diff
@@ -19,34 +19,37 @@ IsotropicRemeshing::IsotropicRemeshing(TriMesh& mesh, const double length, const
     using namespace operations;
     // split
     {
-        OperationSettings<tri_mesh::EdgeSplitAtMidpoint> split_settings{
-            m_position_handle,
-            m_length_max * m_length_max,
-            !m_lock_boundary};
+        OperationSettings<tri_mesh::EdgeSplitAtMidpoint> split_settings;
+        split_settings.position = m_position_handle,
+        split_settings.min_squared_length = m_length_min * m_length_min;
+        split_settings.split_settings.split_boundary_edges = !m_lock_boundary;
+        split_settings.initialize_invariants(m_mesh);
 
         m_scheduler.add_operation_type<tri_mesh::EdgeSplitAtMidpoint>("split", split_settings);
     }
     // collapse
     {
-        OperationSettings<tri_mesh::EdgeCollapseToMidpoint> op_settings{
-            m_position_handle,
-            m_length_min * m_length_min,
```

Comment:
@daniel-zint I changed this from max to min, and this results in a failure in the unit tests, but i still believe the code to be wrong due to the choice of variable names. This is because the variable written to is `hackathon/dev` is `max_squared_length` whereas the variable used to assign is `min * min`.
https://github.com/wildmeshing/wildmeshing-toolkit/blob/b27115ad8dde64f3651af39ff8bc53902d00a2ce/src/wmtk/operations/tri_mesh/EdgeCollapseToMidpoint.hpp#L17 .
For this sort of reason I've removed as much of the bracket initialization as possible in the code - and removed default initialization of these configuration things